src/Crate/PDO/PDOStatement.php

```diff
@@ -202,7 +202,10 @@ public function execute($input_parameters = null)
             if (is_int($parameter) && $zero_based) {
                 $parameter++;
             }
-            $this->bindValue($parameter, $value);
+            // to correctly save arrays and objects, we need to pass
```

Comment:
> Well, it really seems wrong to just fix all parameters as strings. It produces errors where one doesn't expect to find them.

Welcome to PDO :-)